provisioning_templates/user_data/userdata_default.erb

```diff
@@ -76,6 +76,14 @@ runcmd:
 - |
 <%= indent(2) { snippet('saltstack_setup') } %>
 <% end -%>
+- |
+<%=
+  if rhel_compatible && !host_param_false?('cloudinit-networking')
+    indent(2) { snippet 'kickstart_networking_setup' }
+  else
```

Comment:
what about sles and other OS?

Reply:
On my TODO, I am still trying to get this working on RHEL as you can see it's marked as WIP.